aiohttp/client.py

```diff
@@ -150,6 +156,34 @@
     SSLContext = None
 
 
+class _RequestOptions(TypedDict, total=False):
```

Comment:
`total=False` basically means each key-word argument is default (as they all have defaults)